test/klippy/soft_pwm.cfg

```diff
@@ -0,0 +1,14 @@
+[output_pin pwm_pin]
```

Comment:
Thanks for adding this test case.  FWIW, it would be great if there was just one cfg and test file that tested both soft and hard pwm.  (Just define two different output_pin sections in the config.)

Reply:
Agreed.